### DISCUSSION TITLE
Is there any Social Media platform to contact the developer

### DISCUSSION THREAD

**Discussion by Darahaas2001** [SpiralLinux/SpiralLinux-project#17] [General]
Hello , is developer hosting any kind of support channel on Discord / Telegram or any other platforms?

**Comment by ghost** [comment]
On the Spiral Linux website I don't think there is no option for a Telegram or Discord group/room/channel, but I think it would be a good idea to create a group of users using this distribution. It could be created (under my point of view), through some of these three platforms:

- Matrix: You can use a client like SchildiChat or Element.
- Telegram/Nekogram.
- Discord: I have not used Discord for some time, mainly because it is proprietary software, but it has good features such as servers to talk in groups and text and voice channels within the same server.

Matrix and Telegram are free software. Remember that Telegram is open source only the application and not the server part.

If the developer wants and if he has time, it would be a good idea to create a user group on one of these three platforms.

Regarding the distribution, I have reinstalled Spiral Linux, this time with the Budgie desktop and it works very well.  I like this version since Debian does not offer the Budgie desktop during installation.

Sorry for my English in case I made a mistake while writing the message.
Greetings to all!
Un saludo a todos!

**Comment by geckolinux** [comment]
Hi there, thanks for the suggestion. I wouldn't be opposed to creating a Matrix channel. However, the best way to contact me is via the resources listed on the website, in other words, **Github Discussions** or **Issues**.

**Reply by robertocannito** [reply]
I'm happy about Spirallinux and I'm enjoy on all my pcs, gnome DE it's fantastic.
I suggest to open channel also on Facebook and reddit, should be fantastic, github it's ok but sametime it's... :)
